### DISCUSSION TITLE
Welcome to proof_server Discussions!

### DISCUSSION THREAD

**Discussion by nykma** [NextDotID/proof_server#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by javaspeak** [comment]
Hi am JD,

I am a Java Backend Dev who has more recently got into Typescript / Crypto / nestJs / Angular / React

I am trying to bridge UTU Trust tech with next.id DID tech. Am creating a standalone dapp which allows one to manage DID and UTU Trust.

When that is done will integrate the DID into another social media platorm am creating to explore self censorship and how DIDs can reduce software systems being abused by malicious actors.

I do not like when platforms censor comments so am exploring self censorship where the users themselves can decide what they want to see or not.

As I see DID as one of the only way to mitigate malicious actors compromising decentralised software systems I want to promote next.id avatar DID technology and get it used by as many others as possible including myself.

I want to show how social media platforms can greatly reduce malicious distortion of functionality by using next.id avatar DIDs and UTU Trust.

Once I have show cases of how this can be done will contact the bigger boys.

Note this technology can solve similar problems that a twitter tick solves.  I think that the additional burden of building up a next.id avatar DID to improve one's authority online is something people will invest time in. People pay more than $100 / year to have twitter ticks so they must care about their online persona.

**Comment by ElgarMokaya** [comment]
Hi am Elgar,

I am a Java / Typescript Dev who is involved with JD's  next.id / UTU trust integration project.  JD's github handle is javaspeak.

Am pretty young and starting off on my career path.  I am interested in how crypto can help with building trust on the web.

Am looking to make crypto more accessible to everyday web users and show them how crypto is not just an alternative payment system but is about smart usage of private and public keys and the blockchain.

Elgar

**Reply by javaspeak** [reply]
Welcome to the next.id community :-)

**Comment by jebitok-dev** [comment]
Hi, my name is Sharon Jebitok I'm a software developer who has experience working in frontend development and web3. I currently work at Utu with @javaspeak and I'm glad to join the Next.id community. JD has been sharing a lot about Decentralized Identity protocols for quite a while and how we at UTU we can integrate Next.ID and get help with making it easy for our users to access their networks or share their profiles.

**Reply by javaspeak** [reply]
Welcome to the next.id community :-)

**Reply by jebitok-dev** [reply]
Thank you, JD!
